library/src/lib/datetime-picker/datetime-picker.component.ts

```diff
@@ -61,6 +61,9 @@ export class DatetimePickerComponent implements OnInit, ControlValueAccessor {
     @ViewChild(TimeComponent)
     timeComponent;
 
+    @Input()
+    startingDayOfWeek: number = 0;
```

Comment:
Need to ensure this is between 0 and 6, no?

Reply:
There's a note in the docs about 1 being monday, 2 being tuesday etc.  But will limit this to 0-6

Reply:
ideally they should match internally & externally

Reply:
Can you elaborate, not sure what you mean by this

Reply:
like the value being input to the component should be the same as the one being used internally